### DISCUSSION TITLE
关于运行性能测试报错

### DISCUSSION THREAD

**Discussion by shenqimolisha** [LmeSzinc/AzurLaneAutoScript#2132] [Q&A]
从头到尾按照自动安装教程做了，性能测试那里，点击启动，一会儿显示几行都是“unknown host service”，请问我是哪里出错了，怎么解决这个问题。大家有遇到过这个问题吗？ADB我是打开了的，我的是雷电模拟器

**Comment by LmeSzinc** [comment]
这是雷电模拟器专属问题，如果不想折腾建议马上换掉。
https://github.com/LmeSzinc/AzurLaneAutoScript/wiki/Emulator_cn#%EF%B8%8F-%E5%9C%A8%E9%9B%B7%E7%94%B5%E6%A8%A1%E6%8B%9F%E5%99%A8%E4%B8%8A%E8%BF%90%E8%A1%8Calas

**Reply by shenqimolisha** [reply]
好的
